schemas/azure-blob-response.yml

```diff
@@ -0,0 +1,22 @@
+$schema:  http://json-schema.org/draft-04/schema#
+title:    "Azure Shared-Access-Signature Response"
```

Comment:
Included `Blob` and `Table` in the two titles for this schema and the table schema... Afaik golang client uses the titles to generate types :)